mmv1/third_party/terraform/utils/config.go.erb

```diff
@@ -200,30 +201,12 @@ type Config struct {
 	StorageTransferBasePath string
 	BigtableAdminBasePath string
 
-	requestBatcherServiceUsage *RequestBatcher
-	requestBatcherIam          *RequestBatcher
-
-	// start DCLBasePaths
-	// dataprocBasePath is implemented in mm
-	AssuredWorkloadsBasePath string
-	ClouddeployBasePath string
-	CloudResourceManagerBasePath string
+	// dcl
 	ContainerAwsBasePath string
 	ContainerAzureBasePath string
```

Comment:
Do you have context on why these entries need to be manually provided?

Reply:
looks like even though its dcl exclusive resource it needs a custom entry so the data source can have a default. It's a non-ideal behavior.